dashboard/test/ui/features/teacher_tools/teacher_dashboard/teacher_dashboard.feature

```diff
@@ -242,4 +252,4 @@ Feature: Using the teacher dashboard
     And I enter the section code into "input.ui-test-join-section"
     And I click selector "button.ui-test-join-section"
     Then I wait until element ".announcement-notification" is visible
-    And element ".announcement-notification" contains text matching "You are already an instructor for section"
+    And element ".announcement-notification" contains text matching "You are already an instructor for section"
```

Comment:
Do you know what changed here and why? It's probably fine, but just want to check.

Reply:
Honestly, I cannot see a difference myself...